pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/ApexNode.java

```diff
@@ -33,16 +30,6 @@
     Object childrenAccept(ApexParserVisitor visitor, Object data);
 
 
-    /**
-     * Get the underlying AST node.
-     * @deprecated the underlying AST node should not be available outside of the AST node.
-     *      If information is needed from the underlying node, then PMD's AST node need to expose
-     *      this information.
-     */
-    @Deprecated
-    T getNode();
```

Comment:
There is no longer a 1:1 mapping between PMD and the AST. Either this deprecated API needed to be reword or eliminated... the latter seemed like the clear choice.

Reply:
Having access to the underlaying Jorje node made it possible to have the Jorje classes leak into PMDs AST - that's why we deprecated this method. It makes sense to have this removed.

Reply:
👍